src/crd.rs

```diff
@@ -90,3 +112,18 @@ where
     let crd: CustomResourceDefinition = serde_yaml::from_str(T::CRD_DEFINITION)?;
     client.create(&crd).await.and(Ok(()))
 }
+
+/// Waits until CRD of given type `T` is applied to Kubernetes.
+pub async fn wait_ready<T>(client: Client) -> OperatorResult<()>
```

Comment:
This sa also not tied to Crds really....but let's leave it here for now.

The whole Crd trait is not needed anymore I believe since the schemars integration.

Reply:
Ok, I'll leave that one as is now, as I don't have much context. If there are bigger changes coming, let's not waste time on intermediate changes.